CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ BUG FIXES:
 
  * api: added missing namespace field to scaling status GET response object [[GH-8527](https://github.com/hashicorp/nomad/issues/8527)]
  * api: do not allow submission of jobs of type `system` that include task groups with scaling stanzas [[GH-8481](https://github.com/hashicorp/nomad/issues/8481)]
+ * vault: fix batching of expired tokens which could cause large Raft messages when upgrading old (pre-0.11.3) clusters. [[GH-8553](https://github.com/hashicorp/nomad/issues/8553)]
```

Comment:
Looks like we changed capitalization in 0.12.2.  Should this be `api: Fixed ...`.

It feels a bit too much on the implementation side: maybe something like "vault: Fixed a bug where upgrades from pre-0.11.3 that actively use Vault can lead to memory spikes and write large Raft messages."?  Not sure about the Raft memory bit.

Reply:
I like it. Fixing.